src/tests/multihost/alltests/test_auto_mount.py

```diff
@@ -0,0 +1,860 @@
+""" Automation of Auto Mount suite
+:requirement: IDM-31530
```

Comment:
There should be the name of the and it should work. Try to find the child requirement in the polarion instead of the parent where possible.